scripts/train.py

```diff
@@ -45,7 +45,6 @@ def get_args():
     parser.add_argument('--test-interval', type=int, default=10, help='Test interval, one test per n epochs (default: 10)')
     parser.add_argument('--save-interval', type=int, default=10, help='Save interval, one save per n epochs (default: 10)')
     parser.add_argument('--seed', type=int, default=1, help='random seed (default: 1)')
-    parser.add_argument('--distributed-backend', default='ddp', help='Distributed backend: dp, ddp, ddp2')
```

Comment:
This would make it not compatible with previous runs. What is the way used now by pytorch lightning to assign the distributed backed?

Reply:
The trainer class has `strategy` now, which is similar and can also take a training strategy string, like "ddp" for example, but some of possible training strategies changed in name and behavior so just by upgrading to new PL it will behave differently. If you think it's useful I can add the `--distributed-backend` argument back in and set it to a new, reasonable default that is passed to `strategy` in the trainer.

Reply:
These are the strategies:
https://pytorch-lightning.readthedocs.io/en/latest/extensions/strategy.html

I am more concerned that old scripts would directly fail without a why. Either we add a deprecated message or we add a default strategy.

Reply:
You mean using old config files, which include `distributed_backend`, would break? That would just throw an error saying that the argument doesn't exist.

I added a default strategy in the trainer: https://github.com/torchmd/torchmd-net/blob/136e5dd8b10e939bfdc6fe54951a9b1b76ee9d7b/scripts/train.py#L149
I can add the command line argument back in if you want but I would suggest changing the default to "ddp_find_unused_parameters_false".